packages/portalnetwork/src/networks/networkDB.ts

```diff
@@ -166,9 +149,15 @@ export class NetworkDB {
    * Prune DB content to a new radius
    * @param radius node radius
    */
-  async prune(radius: bigint): Promise<void> {
-    for await (const key of this.db.keys({ gte: bigIntToHex(radius) })) {
-      await this.db.del(key)
+  async prune(radius: bigint): Promise<[string, string][]> {
```

Comment:
Should we keep an in memory list of DB keys so we don't have to read from disk just to figure out if a locally stored key can be pruned or not?  I'm thinking just an LRU cache that sits in the networkDB and contains a list of all locally stored keys.  It probably has numerous other performance benefits for down the line (i.e. when we get a FINDCONTENT or OFFER message, we can just check the in memory cache to see if we're storing the content instead of reading from disk)